dotCMS/src/main/java/com/dotmarketing/factories/MultiTreeFactory.java

```diff
@@ -25,691 +16,357 @@
-			
-            Logger.debug(MultiTreeFactory.class, "hibernateUtilSQL:getChildrenClassByOrder\n " + sql);
-			dh.setSQLQuery(sql);
-            Logger.debug(MultiTreeFactory.class, "inode p1:  " + p1.getInode() + "\n");
-            Logger.debug(MultiTreeFactory.class, "inode p2:  " + p2.getInode() + "\n");
-            Logger.debug(MultiTreeFactory.class, "order:  " + order + "\n");
-			
-			dh.setParam(p1.getInode());
-			dh.setParam(p2.getInode());
-
-			return dh.list();
-		}
-		catch (Exception e) {
-            Logger.error(MultiTreeFactory.class, "getChildrenClassByOrder failed:" + e, e);
-			throw new DotRuntimeException(e.toString());
-		}
-	}
-	
-
-	public static java.util.List getParentsOfClassByCondition(Inode p, Class c, String condition) {
-		try {
-			String tableName =  ((Inode) c.newInstance()).getType();
-			HibernateUtil dh = new HibernateUtil(c);
-			
-			String sql = "SELECT {"  + tableName + ".*} from " + tableName + " " + tableName + ", multi_tree multi_tree, inode "+ tableName +"_1_ where multi_tree.child = ? and (multi_tree.parent1 = " + tableName + ".inode or multi_tree.parent2 = " + tableName + ".inode) and " + tableName + "_1_.inode = " + tableName + ".inode and " + condition;
-            Logger.debug(MultiTreeFactory.class, "hibernateUtilSQL:getParentsOfClassByCondition\n " + sql);
-            Logger.debug(MultiTreeFactory.class, "inode:  " + p.getInode() + "\n");
-            Logger.debug(MultiTreeFactory.class, "condition:  " + condition + "\n");
-			dh.setSQLQuery(sql);
-			dh.setParam(p.getInode());
-
-			return dh.list();
-		}
-		catch (Exception e) {
-            Logger.error(MultiTreeFactory.class, "getParentsOfClassByCondition failed:" + e, e);
-			throw new DotRuntimeException(e.toString());
-		}
-	}
-	
-	public static java.util.List getParentsOfClass(Inode p, Class c) {
-		try {
-
-			String tableName =  ((Inode) c.newInstance()).getType();
-			HibernateUtil dh = new HibernateUtil(c);
-			String sql = "SELECT {"  + tableName + ".*} from " + tableName + " " + tableName + ", multi_tree multi_tree, inode "+ tableName +"_1_ where multi_tree.child = ? and (multi_tree.parent1 = " + tableName + ".inode or multi_tree.parent2 = " + tableName + ".inode) and " + tableName + "_1_.inode = " + tableName + ".inode ";
-            Logger.debug(MultiTreeFactory.class, "hibernateUtilSQL:getParentOfClass:\n " + sql+ "\n");
-			dh.setSQLQuery(sql);
-			dh.setParam(p.getInode());
-            Logger.debug(MultiTreeFactory.class, "inode:  " + p.getInode() + "\n");
-			return dh.list();
-		}
-		catch (Exception e) {
-            Logger.error(MultiTreeFactory.class, "getParentsOfClass failed:" + e, e);
-			throw new DotRuntimeException(e.toString());
-		}
-	}	
-	
+    @WrapInTransaction
+    private static void _dbUpsert(final MultiTree mtree) throws DotDataException {
+
+        _dbDelete(mtree);
+        _dbInsert(mtree);
+
+
+
+    }
+
+    private static void _reorder(final MultiTree tree) throws DotDataException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00100)